test/catch2/wrappers/mock_connection.cpp

```diff
@@ -90,7 +90,7 @@ mock_connection::setup_block_manager(WT_SESSION_IMPL *session)
 #if defined(_MSC_VER)
     WT_RET(__wt_os_win(session));
 #else
-    WT_RET(__wt_os_posix(session));
+    WT_RET(__wt_os_posix(session, NULL));
```

Comment:
We need to run the full test suite before merging. I think catch2 tests will segfault on this line

Reply:
[978169b](https://github.com/wiredtiger/wiredtiger/pull/11272/commits/978169b925ae125950f4945d45228b35ef649b1c)